packages/cactus-plugin-htlc-eth-besu-erc20/src/main/typescript/generated/openapi/typescript-axios/.openapi-generator/VERSION

```diff
@@ -0,0 +1 @@
+5.0.0-beta2
```

Comment:
@jordigiam Please make sure to have any globally installed versions of the openapi generator package uninstalled so that the generated files are consistent across the board for everyone (right now when I clone the branch and do a build, I end up with a few changes in the generated files because of the difference)

Reply:
@jordigiam Might be something that we can fix in the scope of another issue: https://app.zenhub.com/workspaces/cactus-zenhub-workspace-6021af681935d30011669aa4/issues/hyperledger/cactus/584